plugin-server/src/worker/ingestion/event-pipeline/enrichExceptionEventStep.ts

```diff
@@ -58,6 +60,12 @@ export function enrichExceptionEventStep(
             if (parsedStack.length > 0) {
                 firstFunction = parsedStack[0].function
             }
+        } else if (exceptionList && exceptionList.length > 0) {
+            const firstException = exceptionList[0]
+            if (firstException.stacktrace) {
+                // TODO: Should this be the last function instead?, or first in app function?
```

Comment:
We should probably consider what the right call is here because it will affect grouping even if it's just in our app. First in-app sounds like the right call to me

Reply:
it should be the closest in app frame from where the exception happened (either where its thrown or captured).
so it depends on how the list is sorted, sentry reverses the list on the backend so better to add a breakpoint and double check.